dsc_lib/src/functions/mul.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::DscError;
+use crate::configure::context::Context;
+use crate::functions::{AcceptedArgKind, Function};
+use serde_json::Value;
+use tracing::debug;
+
+#[derive(Debug, Default)]
+pub struct Mul {}
+
+impl Function for Mul {
+    fn min_args(&self) -> usize {
+        2
+    }
+
+    fn max_args(&self) -> usize {
+        2
+    }
+
+    fn accepted_arg_types(&self) -> Vec<AcceptedArgKind> {
+        vec![AcceptedArgKind::Number]
+    }
+
+    fn invoke(&self, args: &[Value], _context: &Context) -> Result<Value, DscError> {
+        debug!("mul function");
+        let value = args[0].as_i64().unwrap() * args[1].as_i64().unwrap();
```

Comment:
```suggestion
        let value = args[0].as_i64()? * args[1].as_i64()?;
```

You might need to add a new DscError if this doesn't get converted already